pallets/liquid-staking/src/lib.rs

```diff
@@ -214,11 +210,6 @@ pub mod pallet {
     #[pallet::getter(fn exchange_rate)]
     pub type ExchangeRate<T: Config> = StorageValue<_, Rate, ValueQuery>;
 
-    /// Total amount of charged assets to be used as xcm fees.
-    #[pallet::storage]
-    #[pallet::getter(fn insurance_pool)]
-    pub type InsurancePool<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
```

Comment:
InsurancePool should still be put inside LiquidStaking because it's for avoiding being slashed. Before we also use it for paying xcm fees, but we just need to call `add_xcm_fees`

Reply:
@GopherJ It was put inside `ump_transact` https://github.com/parallel-finance/parallel/blob/master/pallets/liquid-staking/src/lib.rs#L998. That's why I placed it inside xcm-helper. 

Do you still want me to put it bank inside liquid-staking

Reply:
yes we should put it back as it's liquid-staking specific

Reply:
liquid staking should just call `add_xcm_fees` in `stake` call